lib/matplotlib/axes/_base.py

```diff
@@ -3196,14 +3201,18 @@ def get_yscale(self):
 
     @docstring.dedent_interpd
     def set_yscale(self, value, **kwargs):
-        """Set the y-axis scale
+        """
+        Set the y-axis scale
 
-        Set the scaling of the y-axis: %(scale)s
+        Parameters
+        ----------
+        value: float
```

Comment:
Missing space before colon, and just like `set_xscale`, I don't think the type is correct.